src/sass/app_2/_common/layout/header.scss

```diff
@@ -101,16 +102,16 @@
                             overflow: hidden;
                             visibility: hidden;
                         }
-                        &.trade:before {
+                        &.menu-links__trade:before {
```

Comment:
Just a heads up, I'll be changing all these external icons to inline SVG components for the dark theme card. So most of these icons will get removed.